### PR TITLE
squeezenet.py for MXNet-master

### PR DESCRIPTION
**Pull request by jinboci**
squeezenet for MXNet-master
Have Not Passed tests for `squeezenet.py` at `test_imagenet_models()` in `test_model_zoo.py` on my local sever:
```
$ python /home/ubuntu/Documents/gluon-cv/tests/model_zoo/test_model_zoo.py
[19:55:44] ../src/storage/storage.cc:199: Using Pooled (Naive) StorageManager for CPU
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.INFO:root:Model file not found. Downloading.
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): apache-mxnet.s3-accelerate.dualstack.amazonaws.com:443
DEBUG:urllib3.connectionpool:https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com:443 "GET /gluon/models/squeezenet1.0-264ba497.zip HTTP/1.1" 200 4660838
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4552/4552 [00:00<00:00, 6706.61KB/s]
F
======================================================================
FAIL: __main__.test_imagenet_models
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/ubuntu/Documents/gluon-cv/tests/model_zoo/test_model_zoo.py", line 155, in test_imagenet_models
    _test_model_list(models, ctx, x)
  File "/home/ubuntu/Documents/gluon-cv/tests/model_zoo/test_model_zoo.py", line 49, in _test_model_list
    net = gcv.model_zoo.get_model(model, pretrained=True, **kwargs)
  File "/home/ubuntu/.local/lib/python3.7/site-packages/gluoncv-0.9.0-py3.7.egg/gluoncv/model_zoo/model_zoo.py", line 403, in get_model
    net = _models[name](**kwargs)
  File "/home/ubuntu/.local/lib/python3.7/site-packages/gluoncv-0.9.0-py3.7.egg/gluoncv/model_zoo/squeezenet.py", line 160, in squeezenet1_0
    return get_squeezenet('1.0', **kwargs)
  File "/home/ubuntu/.local/lib/python3.7/site-packages/gluoncv-0.9.0-py3.7.egg/gluoncv/model_zoo/squeezenet.py", line 138, in get_squeezenet
    tag=pretrained, root=root), ctx=ctx)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/util.py", line 298, in _with_np_shape
    return func(*args, **kwargs)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/util.py", line 482, in _with_np_array
    return func(*args, **kwargs)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/gluon/block.py", line 429, in load_parameters
    self.load_dict(full_dict, ctx, allow_missing, ignore_extra, cast_dtype, dtype_source)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/util.py", line 298, in _with_np_shape
    return func(*args, **kwargs)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/util.py", line 482, in _with_np_array
    return func(*args, **kwargs)
  File "/home/ubuntu/Documents/mxnet/python/mxnet/gluon/block.py", line 474, in load_dict
    name, error_str, _brief_print_list(loaded.keys()))
AssertionError: Parameter 'features.0.weight' is missing in 'file: /home/ubuntu/.mxnet/models/squeezenet1.0-264ba497.params', which contains parameters: 'conv25_bias', 'conv20_bias', 'conv15_weight', ..., 'conv25_weight', 'conv20_weight', 'conv19_bias', 'conv10_bias'. Set allow_missing=True to ignore missing parameters.
-------------------- >> begin captured stdout << ---------------------
Downloading /home/ubuntu/.mxnet/models/squeezenet1.0-264ba497.zip from https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/models/squeezenet1.0-264ba497.zip...

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
root: INFO: Model file not found. Downloading.
urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): apache-mxnet.s3-accelerate.dualstack.amazonaws.com:443
urllib3.connectionpool: DEBUG: https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com:443 "GET /gluon/models/squeezenet1.0-264ba497.zip HTTP/1.1" 200 4660838
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 36 tests in 41.110s

FAILED (SKIP=34, failures=1)
```
Debugging:
```
(Pdb) loaded.keys()
dict_keys(['conv25_bias', 'conv20_bias', 'conv15_weight', 'conv23_weight', 'conv3_weight', 'conv17_weight', 'conv14_weight', 'conv11_bias', 'conv21_weight', 'conv24_bias', 'conv3_bias', 'conv15_bias', 'conv23_bias', 'conv6_weight', 'conv14_bias', 'conv22_bias', 'conv19_weight', 'conv17_bias', 'conv24_weight', 'conv0_weight', 'conv7_weight', 'conv4_weight', 'conv2_bias', 'conv9_weight', 'conv12_weight', 'conv1_weight', 'conv8_weight', 'conv16_weight', 'conv18_weight', 'conv16_bias', 'conv7_bias', 'conv12_bias', 'conv4_bias', 'conv11_weight', 'conv9_bias', 'conv21_bias', 'conv8_bias', 'conv22_weight', 'conv0_bias', 'conv18_bias', 'conv5_bias', 'conv13_weight', 'conv1_bias', 'conv10_weight', 'conv13_bias', 'conv5_weight', 'conv2_weight', 'conv6_bias', 'conv25_weight', 'conv20_weight', 'conv19_bias', 'conv10_bias'])
(Pdb) params.keys()
dict_keys(['features.0.weight', 'features.0.bias', 'features.3.0.0.weight', 'features.3.0.0.bias', 'features.3.1.0.0.weight', 'features.3.1.0.0.bias', 'features.3.1.1.0.weight', 'features.3.1.1.0.bias', 'features.4.0.0.weight', 'features.4.0.0.bias', 'features.4.1.0.0.weight', 'features.4.1.0.0.bias', 'features.4.1.1.0.weight', 'features.4.1.1.0.bias', 'features.5.0.0.weight', 'features.5.0.0.bias', 'features.5.1.0.0.weight', 'features.5.1.0.0.bias', 'features.5.1.1.0.weight', 'features.5.1.1.0.bias', 'features.7.0.0.weight', 'features.7.0.0.bias', 'features.7.1.0.0.weight', 'features.7.1.0.0.bias', 'features.7.1.1.0.weight', 'features.7.1.1.0.bias', 'features.8.0.0.weight', 'features.8.0.0.bias', 'features.8.1.0.0.weight', 'features.8.1.0.0.bias', 'features.8.1.1.0.weight', 'features.8.1.1.0.bias', 'features.9.0.0.weight', 'features.9.0.0.bias', 'features.9.1.0.0.weight', 'features.9.1.0.0.bias', 'features.9.1.1.0.weight', 'features.9.1.1.0.bias', 'features.10.0.0.weight', 'features.10.0.0.bias', 'features.10.1.0.0.weight', 'features.10.1.0.0.bias', 'features.10.1.1.0.weight', 'features.10.1.1.0.bias', 'features.12.0.0.weight', 'features.12.0.0.bias', 'features.12.1.0.0.weight', 'features.12.1.0.0.bias', 'features.12.1.1.0.weight', 'features.12.1.1.0.bias', 'output.0.weight', 'output.0.bias'])
```

Changes due to mxnet-master's different implementations from 1.7:
- `nn.HybridSequential` on mxnet-master does not accept the parameter `prefix`
- No `name_scope` defined in class `HybridSequential` `HybridBlock` `Block` on mxnet-master (block.py)
- `HybridConcatenate`: https://github.com/apache/incubator-mxnet/pull/19046

### CODE REVIEW

File: gluoncv/model_zoo/model_store.py [hunk 1]
```diff
@@ -37,8 +37,8 @@
     ('ecdde35339c1aadbec4f547857078e734a76fb49', 'resnet50_v2'),
     ('18e93e4f48947e002547f50eabbcc9c83e516aa6', 'resnet101_v2'),
     ('f2695542de38cf7e71ed58f02893d82bb409415e', 'resnet152_v2'),
-    ('264ba4970a0cc87a4f15c96e25246a1307caf523', 'squeezenet1.0'),
-    ('33ba0f93753c83d86e1eb397f38a667eaf2e9376', 'squeezenet1.1'),
+    # ('264ba4970a0cc87a4f15c96e25246a1307caf523', 'squeezenet1.0'),
+    # ('33ba0f93753c83d86e1eb397f38a667eaf2e9376', 'squeezenet1.1'),
```
Comment (jinboci) [inline gluoncv/model_zoo/model_store.py:41]: Avoid loading pretrained models

**Comment by mli** [comment]
Job PR-1521-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1521/2/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1521-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1521/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-1521-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1521/5/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**